tests/linux/clean/kibana/ac96ceb8-4399-4191-af1d-4feeac1f1f46_108.json.simple

```diff
@@ -0,0 +1,11 @@
+# linux/clean/kibana/ac96ceb8-4399-4191-af1d-4feeac1f1f46_108.json: medium
```

Comment:
I think we had a JSON regression here; We previously skipped JSON files unless they matched a particular pattern indicative of NPM packaging:

https://github.com/chainguard-dev/malcontent/blob/8c381b30bd77d222e8e3e434c908670db41c2fea/pkg/programkind/programkind.go#L195

Reply:
Reverted in [`81c9d1f` (#898)](https://github.com/chainguard-dev/malcontent/pull/898/commits/81c9d1f8c1f62e1728da01a104331e92dbd24af7).